### PR TITLE
Adding source: affaldonline_dk

### PR DESCRIPTION
**Pull request by danielklejnstrup**
./test_sources.py -s middelfart_dk -i -l
Testing source middelfart_dk ...
  found 1 entries for test
    2024-06-20 : Dagrenovation [None]

./test_sources.py -s assensforsyning_dk -i -l
Testing source assensforsyning_dk ...
  found 1 entries for test
    2024-06-24 : Rest/Mad [None]

./test_sources.py -s silkeborgforsyning_dk -i -l
Testing source silkeborgforsyning_dk ...
  found 8 entries for test
    2024-06-21 : Mad-/Rest [None]
    2024-06-28 : Mad-/Rest [None]
    2024-06-28 : Plast/MDK/Glas/Metal/Papir/Pap [None]
    2024-07-05 : Mad-/Rest [None]
    2024-07-12 : Mad-/Rest [None]
    2024-07-19 : Mad-/Rest [None]
    2024-07-19 : Plast/MDK/Glas/Metal/Papir/Pap [None]
    2024-07-26 : Mad-/Rest [None]

### CODE REVIEW

**Comment by danielklejnstrup** [comment]
Actually, I have a general Affaldonline source in the works, so I'll hold off on adding more individual sources for now...
Come to think of it - how do you handle discontinued sources? Simply removing them could break someones use, and there's no way to know about usage?

**Comment by danielklejnstrup** [comment]
Testing source affaldonline_dk ...
  found 1 entries for aeroe
    2024-06-17 : Mad/Rest [None]
  found 3 entries for favrskov
    2024-06-20 : Rest-/madaffald [None]
    2024-06-20 : Papir/pap og tekstiler [None]
    2024-06-27 : Plast/mad- og drikkekartoner og glas/metal [None]
  found 5 entries for silkeborg
    2024-06-26 : Mad-/Rest [None]
    2024-07-05 : Plast/MDK/Glas/Metal/Papir/Pap [None]
    2024-07-10 : Mad-/Rest [None]
    2024-07-24 : Mad-/Rest [None]
    2024-07-26 : Plast/MDK/Glas/Metal/Papir/Pap [None]

**Comment by danielklejnstrup** [comment]
I'm inclined to remove the not-yet-pulled individual sources.
I've seen you discourage use of several test cases, but here I think at least one test from each parser should be used?
The function select_test_cases can be called with different modes - a random variant would ensure that a change in a single municipality's layout will be caught.

**Comment by 5ila5** [comment]
I think If you have the generic source that works for all the other ones, you do not need a dedicated source for every municipality. You can add more municipalities to the README.md by using the EXTRA_INFO parameter ([more information](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/contributing_source.md#update-links-in-readmemd-and-infomd))

File: doc/source/affaldonline_dk.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+              values: "Nørregade|1||||8382|Hinnerup|6443|108156|0"
+```
+
+### Configuration Variables
+
+**values**  
+_(String) (required)_
+
+A string that includes the street name, house number, postal code, city name, and some numbers that seems to be internal affaldonline references. 
+
```
Comment (5ila5) [inline doc/source/affaldonline_dk.md:24]: please add documentation for the municipality parameter

**Comment by danielklejnstrup** [comment]
That should be it?
What do you think about testing all the municipalities?

Testing source affaldonline_dk ...
  found 2 entries for aeroe
    2024-06-24 : Mad/Rest [None]
    2024-06-24 : Papir/Pap/Glas [None]
  found 1 entries for assens
    2024-06-25 : Rest/Mad [None]
  found 4 entries for langeland
    2024-07-01 : Bioaffald [None]
    2024-07-01 : Restaffald [None]
    2024-07-01 : Plast/Drikkekarton [None]
    2024-07-01 : Tekstilaffald [None]
  found 1 entries for middelfart
    2024-06-19 : Dagrenovation [None]
  found 1 entries for rebild
    2024-06-20 : Genbrug [None]
  found 2 entries for vejle
    2024-06-18 : Genanvendeligt [None]
    2024-06-18 : Dagrenovation [None]
  found 3 entries for favrskov
    2024-06-20 : Rest-/madaffald [None]
    2024-06-20 : Papir/pap og tekstiler [None]
    2024-06-27 : Plast/mad- og drikkekartoner og glas/metal [None]
  found 5 entries for silkeborg
    2024-06-26 : Mad-/Rest [None]
    2024-07-05 : Plast/MDK/Glas/Metal/Papir/Pap [None]
    2024-07-10 : Mad-/Rest [None]
    2024-07-24 : Mad-/Rest [None]
    2024-07-26 : Plast/MDK/Glas/Metal/Papir/Pap [None]

**Comment by danielklejnstrup** [comment]
Just saw the 1.49 release including my other sources.
What do you think about removing them? -- can it be announced in the next release, that they are replaced by affaldonline_dk?

**Comment by 5ila5** [comment]
I think remove them is fine, I don't think there will be a lot of people affected when remove such a new source.

I added the not yet documented `default_params` to your EXTRA_INFO which will be used to set default parameters when selecting the region in the upcoming GUI (config flow) configuration (#2068)

**Pull request merged**